### ISSUE TITLE
notice: lowercase letter "p" does not display correctly using callout syntax

### ISSUE THREAD

**Issue by BraidenPsiuk** [McShelby/hugo-theme-relearn#965]
If you create a notice using callout syntax, and set the title to something that ends with the lowercase letter "p", it won't render the `p`. Uppercase `P` works.

## Example

```
> [!note] Heads up
> The letter "p" does not display properly in the title.
```

![notice-issue](https://github.com/user-attachments/assets/a617d56a-eee9-44f1-998f-82decc25efe8)

This doesn't seem to be a problem when using shortcode syntax.

**Comment by McShelby** [comment]
Thanks for posting this. Sadly, after a quick check, this seems to be a Hugo issue. I'll post an issue in Hugos bugtracker later.

**Comment by BraidenPsiuk** [comment]
Awesome, thanks! Honestly thought this was something silly I was doing and it wasn't going to be easily reproducible.

**Comment by McShelby** [comment]
For reference: https://github.com/gohugoio/hugo/issues/13119

**Comment by McShelby** [comment]
Closed in favor of uplink issue.

**Comment by McShelby** [comment]
This was fixed in Hugo 0.140.0, released yesterday.
https://github.com/gohugoio/hugo/releases/tag/v0.140.0
